src/TgSharp.Core/DataCenter.cs

```diff
@@ -1,20 +1,25 @@
 namespace TgSharp.Core
 {
-    internal class DataCenter
+    public class DataCenter
     {
-        internal DataCenter (int? dcId, string address, int port)
+        public DataCenter()
```

Comment:
given that this constructor will only be used when serializing/deserializing, I think we should add an `[Obsolete("do not use, this ctor is public only for serialization"]` attribute